pages/glossary/whitespace.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Whitespace
+key: whitespace
+---
+
+Whitespace are characters that have the Unicode "White_Space" property in the [Unicode properties list](https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt).
```

Comment:
Perhaps "Whitespace comprises characters..."

"Whitespace are characters..." feels like incorrect grammar.

Reply:
Agreed with Emma's 4 posts here :-)

Reply:
Not sure why that posted 4 times ... sorry, wasn't trying to drive home a point or anything.

Reply:
Duplicates removed

Reply:
@EmmaJP and @WilcoFiers , agree that it could need some improvement.

However, I had to look up the word "comprise" to make sure what it meant, and it's on the "Don't say" list for plain language on https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/.

Could we use e.g. is made up of, consist of, includes, or another synonym (e.g. from https://www.thesaurus.com/browse/comprise)?

Maybe native English speaker @Brynanders also has an opinion on this?

Reply:
@brynanders suggested just shortening it to "Characters that have the Unicode "White_Space" property", which I think is much cleaner, so I'll go with that until someone objects :)